test/test.js

```diff
@@ -95,6 +95,58 @@ describe('exiv2', function(){
       });
     });
   });
+  
+  describe('.setUtf8ImageTags()', function(){
```

Comment:
Since this isn't adding a new function can we just have the existing setImageTags tests write UTF and ASCII values?